lib/salesforce/donation_validator.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'salesforce/result'
+
+module Salesforce
+  class DonationValidator
```

Comment:
Just an aside, you might want to consider a generic ruby validation library to handle your validations in the future, for example: https://github.com/dry-rb/dry-validation

Reply:
This gem was made with Rails in mind, we have our own validations that are specific to our use case. I think you may possibly have a slight case of gemitis. :wink:

Reply:
No, not all. It was made by Piotr Solnica specifically to have a validation library that wasn't based on Rails. Piotr is a Hanami contributor and also the author of ROM, which is an ActiveRecord competitor.

Let me know how it's going writing all your own code, with a team of two ;)

Reply:
This is a blog post from Piotr from last year. His gems don't support Rails anymore: http://solnic.eu/2016/05/22/my-time-with-rails-is-up.html